### PR TITLE
[12.0][MIG] account_cost_center

### PR DESCRIPTION
**Pull request by astirpe**
Depends on:

- [x] https://github.com/OCA/server-tools/pull/1529

### CODE REVIEW

File: account_cost_center/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import models
```
Comment (jesusVMayor) [inline account_cost_center/__init__.py:3]: Tested, works fine, but when you install it you need to update the account module to recreate the account.invoice.report database view. You could add a post_init_hook

```suggestion
from . import models
from odoo import api, SUPERUSER_ID
def recreate_invoice_report_view(cr, registry):
    env = api.Environment(cr, SUPERUSER_ID, {})
    env['account.invoice.report'].init()
```

File: account_cost_center/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    'data': [
+        'security/ir.model.access.csv',
+        'security/account_cost_center_security.xml',
+        'views/account_cost_center.xml',
+        'views/account_move.xml',
+        'views/account_move_line.xml',
+        'views/account_invoice.xml',
+        'views/account_invoice_report.xml',
+    ],
+    'installable': True,
```
Comment (jesusVMayor) [inline account_cost_center/__manifest__.py:25]:
```suggestion
    'installable': True,
    'post_init_hook': 'recreate_invoice_report_view',
```

**Comment by astirpe** [comment]
@jesusVMayor I'm trying locally. When I install or uninstall the module, the `init()` of account.invoice.report is automatically triggered, without any need of a `post_init_hook`. Could you check again?

**Review by jesusVMayor — APPROVED** [review]
Yes, seems to work right, i dont know what i made the last time. Sorry for the mistake.

**Comment by astirpe** [comment]
No problem @jesusVMayor

**Comment by GSLabIt** [comment]
is this ready to merge?

**Comment by astirpe** [comment]
Not yet. Reviews are missing.

File: oca_dependencies.txt [hunk 3]
```diff
@@ -1 +1,2 @@
 server-ux
+server-tools https://github.com/akretion/server-tools/ 12.0-mig-base_view_inheritance_extension
```
Comment (StefanRijnhart) [inline oca_dependencies.txt]: You can replace with the OCA branch now that the dependent module is merged.

**Comment by astirpe** [comment]
@StefanRijnhart  done, thanks!

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**